pkg/clusteragent/admission/mutate/tagsfromlabels/tags_test.go

```diff
@@ -273,7 +275,8 @@ func TestGetAndCacheOwner(t *testing.T) {
 	kubeObj := newUnstructuredWithSpec(map[string]interface{}{"foo": "bar"})
 	owner := newOwner(kubeObj)
 	wmeta := fxutil.Test[workloadmeta.Component](t, core.MockBundle(), workloadmetafxmock.MockModule(workloadmeta.NewParams()))
-	webhook := NewWebhook(wmeta, autoinstrumentation.GetInjectionFilter())
+	datadogConfig := fxutil.Test[config.Component](t, core.MockBundle())
+	webhook := NewWebhook(wmeta, datadogConfig, autoinstrumentation.GetInjectionFilter())
```

Comment:
Curious. Whats the difference between mocking the config component via:

`datadogConfig := fxutil.Test[config.Component](t, core.MockBundle())`

and

`mockConfig := configmock.New(t)` from https://github.com/DataDog/datadog-agent/blob/3270886c03d9b4a94598a0a8230f44a0b2b9c294/pkg/config/mock/mock.go#L27

Reply:
Also I suppose that `configmock.New(t)` can problematic with tests parallelization.